console/src/main/webapp/index.html

```diff
@@ -10,6 +10,10 @@
     <!-- endbower -->
     <link href="dist/hawkular-console.css?v=${hawkular.console.timestamp}" rel="stylesheet">
 
+    <!-- BTM INTEGRATION START -->
```

Comment:
This should probably be before hawkular-console.css in order not to override any hawkular specific styles